DBSCE/Overview.md

```diff
@@ -0,0 +1,221 @@
+
+One device registration client can manage multiple devices on the same physical device. There also can be multiple device registration clients on the same device. The device registration client can be owned and supported by:
+
+- Operating system - the device gets registered when the OS is installed.
+- Browser - the device gets registered when the browser is installed.
+- Device management software (MDM provider) - the device gets registered when the MDM is enrolled.
+- Third-party software vendor - the device gets registered according to the vendor rules.
+
+DBSC(E) aims to support most of these scenarios. It does not define the device registration protocol and is only concerned with the keys generated in a "clean room" state and the management of the generated keys to prove device binding.
+
+
+### Local Key Helper
+
+DBSC(E) introduces the concept of `Local Key Helper`.
+
+**Local Key Helper** is an integral part of the the **Device Registration Client**, a software interface responsible for the DBSC Key management. *Local key helper* can be public or private and is expected to be either shipped as a part of a given enterprise framework (with the IdP/OS) or can be installed by a provider in compliance with the protocol expanded below.
+
+From the deployment point of view there are two types of local key helpers: _well-known_(_private_) and _third party_(_public_)
+
+- _Public local key helper_ or _third party_: Can be accessed by any Identity Provider (IdP). Typically owned by a provider different from the IdP, communicates with the IdP through as defined in DBSC(E) protocol. A third party local key helper has a special deployment mechanism. We expect to loosely define the API and deployment mechanism in the upcoming specification, and will leave the specific implementation details to the vendors. However, we have provided examples of how this can look for a few platforms [below](#platform-examples).
+- _Private local key helper_ or _well known_ : Can be specific to an IdP. Typically owned by the IdP and will have a private protocol to communicate with the IdP. Comes with either OS or built into the browser. It assumed that well-known key helpers are **trusted and enabled by default** in a browser and/or a given IDP. A browser knows how to activate a well-known key helper.
+
+The Local Key Helper is responsible for:
+
+- Generation of the binding key and producing binding statements (see below)
+- Producing signatures with the binding key
+- Cleanup of the binding key and its artifacts (when the user clears the browser session or the key is unused for a long time).
+
+#### Platform Examples
```

Comment:
Per above comment, suggesting to change to "Platform requirements" rather than examples

Reply:
Done